democracy_club/templates/hermes/post_detail.html

```diff
@@ -22,23 +22,14 @@
       <h1>{{ post.subject }}</h1>
       <div class="byline">
         <address class="author">
-          By
-          {% if post.author.username == "symroe" %}
-            <a rel="author" href="https://twitter.com/symroe">
-              {{post.author.first_name}}
-              {{post.author.last_name}}
-            </a>
-          {% elif post.author.first_name == "Joe" %}
-            <a rel="author" href="https://twitter.com/j0e_m">
-              {{post.author.first_name}}
-              {{post.author.last_name}}
-            </a>
+        By
+        {% for author in post.author.all %}
+          {% if post.author.count == 1 %}
+            {% include 'linked_authors.html' with author=author%}
           {% else %}
-          <a rel="author" href="{% url 'team' %}#{{post.author.first_name}}">
-            {{post.author.first_name}}
-            {{post.author.last_name}}
-          </a>
+            {% if forloop.last %} and {% else %}{% if not forloop.first %}, {% endif %}{% endif %}{% include 'linked_authors.html' with author=author %}
```

Comment:
Currently, a user can be created in admin without adding a first or last name. This will cause an empty `,` if a user without a first or last name is added where there are multiple users. 
We could:

- make first and/or last name required fields when a user is created or
- default to a username when the first and last name doesn't exist. 

The latter would cause an issue with the logic that relies on `is_staff` to match the team page anchor.

Reply:
Good point. The former is tricky because it would mean changing Django core somehow. I think the latter is the better option, and we can just accept it'll break the link (or we could move to using the username in the IDs on the team page?)